release-1.8/release_notes_draft.md

```diff
@@ -211,19 +229,15 @@ kind.
 
 #### Cluster Discovery/Bootstrap
 
-* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new members to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
-
-#### Addons
-* [alpha] A new system addon manager is available that is aiming to improve the downsides of the existing `kube-addons.sh` manager. ([#18](https://github.com/kubernetes/features/issues/18), [@justinsb](https://github.com/justinsb))
+* [beta] The authentication and verification mechanism called Bootstrap Tokens has been added to the core API, which can be used to easily add new node identities to a cluster. ([#130](https://github.com/kubernetes/features/issues/130), [@luxas](https://github.com/luxas), [@jbeda](https://github.com/jbeda)).
```

Comment:
The authentication and verification mechanism called Bootstrap Tokens has been added to the core API. Use Bootstrap Tokens to add new node identities to a cluster easily.

Reply:
fixed